torsions/TorsionFitModel.py

```diff
@@ -28,34 +33,56 @@ def __init__(self, param, stream, frags):
 
             for m in multiplicities:
                 name = p[0]+'_'+p[1]+'_'+p[2]+'_'+p[3]+'_' + str(m) + '_K'
-                k = Uniform(name, lower=0, upper=20, value=0)
+                k = pymc.Uniform(name, lower=0, upper=20, value=0)
                 for i in range(len(param.dihedral_types[p])):
                     if param.dihedral_types[p][i].per == m:
                         multiplicity_bitstrings[torsion_name] += 2**(m-1)
-                        k = Uniform(name, lower=0, upper=20, value=param.dihedral_types[p][i].phi_k)
+                        k = pymc.Uniform(name, lower=0, upper=20, value=param.dihedral_types[p][i].phi_k)
                         break
 
                 self.pymc_parameters[name] = k
 
                 name = p[0]+'_'+p[1]+'_'+p[2]+'_'+p[3]+'_' + str(m) + '_Phase'
-                phase = DiscreteUniform(name, lower=0, upper=1, value=0)
+                phase = pymc.DiscreteUniform(name, lower=0, upper=1, value=0)
                 for i in range(len(param.dihedral_types[p])):
                     if param.dihedral_types[p][i].per == m:
                         if param.dihedral_types[p][i].phase == 0:
-                            phase = DiscreteUniform(name, lower=0, upper=1, value=0)
+                            phase = pymc.DiscreteUniform(name, lower=0, upper=1, value=0)
                             break
                         if param.dihedral_types[p][i].phase == 3.141592653589793:
-                            phase = DiscreteUniform(name, lower=0, upper=1, value=1)
+                            phase = pymc.DiscreteUniform(name, lower=0, upper=1, value=1)
                             break
 
                 self.pymc_parameters[name] = phase
 
         for torsion_name in multiplicity_bitstrings.keys():
             name = torsion_name + '_multiplicity_bitstring'
-            bitstring = DiscreteUniform(name, lower=0, upper=63, value=multiplicity_bitstrings[torsion_name])
+            bitstring = pymc.DiscreteUniform(name, lower=0, upper=63, value=multiplicity_bitstrings[torsion_name])
             self.pymc_parameters[name] = bitstring
 
-        self.frags = frags
+        self.pymc_parameters['log_sigma'] = pymc.Uniform('log_sigma', lower=-10, upper=0, value=np.log(0.01))
```

Comment:
Did you pick this range for `log_sigma` based on some estimates of the maximum error?  It may be reasonable to expand the `upper` end to, say, +3.